lib/parallel_tests/grouper.rb

```diff
@@ -28,7 +28,39 @@ def in_even_groups_by_size(items, num_groups, options = {})
           raise 'Number of isolated processes must be less than total the number of processes'
         end
 
-        if isolate_count >= 1
+        if options[:specify_groups]
+          specify_test_process_groups = options[:specify_groups].split('|')
```

Comment:
prefer a single data structure:
```suggestion
          specify_test_process_groups = options[:specify_groups].split('|').map { |g| g.split(',') }
```

Reply:
We need them separate because we need to have a count of the number of groups, and to be able to differentiate the groups.  We only will use `all_specified_tests` to extract them from the items variable